rebar.config

```diff
@@ -12,18 +12,15 @@
                locals_not_used,
                deprecated_function_calls, deprecated_functions]}.
 
-{eunit_opts, [
-verbose
-%     no_tty,  %% This turns off the default output, MUST HAVE
-%     {report, {eunit_progress, [colored, profile]}} %% Use `profile' to see test timing information
-     %% Uses the progress formatter with ANSI-colored output
-     ]}.
+
+{eunit_opts, [no_tty, {report, {unite_compact, []}}]}.
```

Comment:
Maybe reinstate the comment about why we have `no_tty` set?  Also what does `unite_compact` specify?